src/v0/destinations/klaviyo/transform.js

```diff
@@ -42,61 +54,84 @@ const { ConfigurationError, InstrumentationError } = require('../../util/errorTy
  */
 const identifyRequestHandler = async (message, category, destination) => {
   // If listId property is present try to subscribe/member user in list
-  if (!destination.Config.publicApiKey || !destination.Config.privateApiKey) {
-    if (!destination.Config.publicApiKey) {
-      throw new ConfigurationError('Public API Key is a required field for identify events');
-    } else {
-      throw new ConfigurationError('Private API Key is a required field for identify events');
-    }
+  const { privateApiKey, enforceEmailAsPrimary } = destination.Config;
+  if (!privateApiKey) {
+    throw new ConfigurationError('Private API Key is a required field for identify events');
```

Comment:
What if the customer is not sending any identify calls? It is possible customer is sending track calls only. Then privateApiKey is not a required field, right?

Reply:
It is required for all calls now.

Reply:
Then update the error message accordingly.

Reply:
Updated